templates/423.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% set active_page = "support" %}
+
+{% block content %}
+<div class="container">
+    <div class="row" style="margin-top:50px;">
+        <div class="col-md-12">
+            <h1><strong>Locked</strong></h1>
```

Comment:
Could we change the header from "Locked" to say a more descriptive title, such as "Unable to Access Project"?